pkgs/applications/networking/ipfs-cluster/default.nix

```diff
@@ -2,15 +2,15 @@
 
 buildGoModule rec {
   pname = "ipfs-cluster";
-  version = "1.0.1";
+  version = "1.0.2";
 
-  vendorSha256 = "sha256-V+fqyrol+hXjjaCBlAs6f7FeqBqa2jTmMO2bvb6HfgY=";
+  vendorSha256 = "sha256-4pCJnQ/X5bvlgyHcRVZ8LyOexaKmz+1xAntMpZCpvd0=";
 
   src = fetchFromGitHub {
     owner = "ipfs";
```

Comment:
The repo is moved to https://github.com/ipfs-cluster/ipfs-cluster/, will be nice to update this as well.

Reply:
This wasn't yet the case, when I created the PR, so thank you for catching that.